src/main/java/de/zalando/ep/zalenium/matcher/DockerSeleniumCapabilityMatcher.java

```diff
@@ -48,22 +51,6 @@ public boolean matches(Map<String, Object> nodeCapability, Map<String, Object> r
             return false;
         }
 
-        // We do this because the starter node does not have the browser versions when Zalenium starts
```

Comment:
@diemol Do you know if this code is necessary to not?

Reply:
Yes and no, the idea of it is to reject as fast as possible a test request with a browser version we don't provide. 

We need to test what happens if I want to run a test with Chrome version 60, and we provide version 66. The request should be rejected since it should not match any nodes.

In the end, this is a minor thing since we can find ways to tweak it.